staff_features/shared/step_definitions.rb

```diff
@@ -24,8 +24,14 @@
 
 When 'the user clicks on {string} in the dropdown menu' do |string|
   within '.dropdown-menu' do |dropdown_menu|
-    elements = dropdown_menu.all(:xpath, "//*[contains(text(), '#{string}')]")
-    elements[1].click
+    elements = dropdown_menu.all(:xpath, ".//*[contains(text(), '#{string}')]")
```

Comment:
Minor point for robustness -- if a menu had options like 'Copy' 'Copy All'  and this test was executed with 'Copy' in {string} (as a contrived example) I think this would result in both menu options being clicked. It might be better to test for exact match on the menu item string, unless that behavior is intended.

Reply:
Fixed.